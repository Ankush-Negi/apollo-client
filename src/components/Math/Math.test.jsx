import React from 'react';
import { mount } from 'enzyme';
import MathDemo from './Math';

const props = {
  first: 10,
  second: 20,
  operator: '+',
};

describe('BMath Component', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = mount(<MathDemo {...props} />);
  });
  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
  it('render math component: ', () => {
    const result = wrapper.find('MathDemo');
    expect(result).toEqual({});
  });
});
