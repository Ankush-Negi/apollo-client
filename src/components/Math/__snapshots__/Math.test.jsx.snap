// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BMath Component should render correctly 1`] = `
<MathDemo
  first={10}
  operator="+"
  second={20}
>
  <h4>
    10
     
    +
     
    20
     
    =
     
    30
  </h4>
</MathDemo>
`;
